src/Prometheus/Storage/InMemory.php

```diff
@@ -10,8 +10,19 @@
 class InMemory implements Adapter
 {
 
+    /**
+     * @var mixed[]
+     */
     private $counters = [];
```

Comment:
Should be a Counter[]

Reply:
So according to lines 221-224 this is not actually the case:
```
            $this->counters[$metaKey] = [
                'meta' => $this->metaData($data),
                'samples' => [],
            ];
```

This is the same for gauges and histograms. The Counter/Gauge/Histogram classes are not even imported in the `use` section.

Reply:
mmh yep you are right. I personally think in the future we should cleanup this mix of using arrays within arrays on the one side and arrays using objects on the other side.